### DISCUSSION TITLE
Unable to add signatures

### DISCUSSION THREAD

**Discussion by RKNirvana** [JakubMelka/PDF4QT#209] [Q&A]
Hello,

Is there any specific method need to be updated while adding Signature fields ?. I tried adding signature (SVG image) and also by adding text, both are not saving to the original file when saved.

**Comment by JakubMelka** [comment]
Hello, @RKNirvana, could you please attach sample PDF file? I will try to sign it.

**Reply by YESNLars** [reply]
Do you mean with signing: adding a signature via svg or something to do with signing digitally? And I can attach the sample pdf, if you put the .svg file into the "Handtekening" box, do you need to do anything else to save it (except saving as/overwriting the file)? Cuz there may be a step such as signing digitally to finish
[test_signing.pdf](https://github.com/user-attachments/files/25660429/test_signing.pdf)

**Reply by YESNLars** [reply]
So I believe we both (@RKNirvana and me) misunderstood the feature and didn't finish the signing process via that signing icon checkmark next to the normal signing icon

**Comment by JakubMelka** [comment]
No answer, closing the discussion.
